src/Bicep.Core/PrettyPrintV2/SyntaxLayouts.Operators.cs

```diff
@@ -77,13 +78,27 @@ private Document Bracket(SyntaxBase openSyntax, Func<IEnumerable<Document>> item
                 .TrimNewlines()
                 .CollapseNewlines(onHardLine: this.ForceBreak)
                 .Concat(danglingComments)
-                .SeparateBy(separator);
+                .SeparateBy(separator)
+                .ToImmutableArray();
 
-            if (!items.Any())
+            if (items.Length is 0)
```

Comment:
is?  Does that compile to a relational pattern?  Why not ==?

Reply:
It gets compiled to `==`. This is just a personal preference because I find the pattern matching syntax neat (e.g. `if (foo.Length is 1 or 2)` is more readable than `if (foo.Length == 1 || foo.Length == 2)`.

Reply:
Guess I'm not sure yet, although as long as it compiles to ==...